RUL-91: add enabled key info on rule import (#1508)
akeneo/pim-docs: manipulate_pim_data/rule/general_information_on_rule_format.rst

Therefore, a 90-priority rule  will be executed before 0-priority ones. If two rules have the same priority,
they will be executed in a "technical" order. (database reading order)

> A status can be defined through the `enabled` key. When the value is `false` the rule cannot be executed neither from cronjob nor manually.

Reviewer: ```suggestion
A status can be defined with the `enabled` key. When the value is `false`, the rule cannot be executed from cronjob or manually.
```

> In the rule import this key is not mandatory, the rule is enabled by default if it's not specified.

Reviewer: ```suggestion
In the rule import, this key is not mandatory, the rule is enabled, set to true, by default if it's not specified.
```
